go.sum

```diff
@@ -27,6 +27,8 @@ github.com/cyphar/filepath-securejoin v0.4.1 h1:JyxxyPEaktOD+GAnqIqTf9A8tHyAG22r
 github.com/cyphar/filepath-securejoin v0.4.1/go.mod h1:Sdj7gXlvMcPZsbhwhQ33GguGLDGQL7h7bg04C/+u9jI=
 github.com/databricks/databricks-sdk-go v0.94.0 h1:+ITzL/O6/8FkHBJ4oWj1o8PkZ9GPozqydpXJRjsdxzw=
 github.com/databricks/databricks-sdk-go v0.94.0/go.mod h1:hWoHnHbNLjPKiTm5K/7bcIv3J3Pkgo5x9pPzh8K3RVE=
+github.com/databricks/databricks-sdk-go v0.95.0 h1:ZehbIygQgK3yeKEJIiichtftwPmiYVb4SrqBukJGbFU=
+github.com/databricks/databricks-sdk-go v0.95.0/go.mod h1:hWoHnHbNLjPKiTm5K/7bcIv3J3Pkgo5x9pPzh8K3RVE=
```

Comment:
This would need to be cleaned up (previous go SDK version still exists) 0.94.0, line 28 above. Please run go mod tidy.